### PR TITLE
ledger report reconcile

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/models/ledgerreport.go [hunk 1]
```diff
@@ -91,14 +91,25 @@ func (lr *LedgerReport) IsActorAllowedTo(tx *pop.Connection, actor User, perm Pe
 // ConvertToAPI converts a LedgerReport to api.LedgerReport
 func (lr *LedgerReport) ConvertToAPI(tx *pop.Connection) api.LedgerReport {
```
Comment (Baggerone) [inline application/models/ledgerreport.go:92]: (This is an older issue, but maybe it's worth changing now.)

The matching method for LedgerReports (on line 34) returns an interface.  Is that necessary? I think best-practice in Go is to accept interfaces but return concrete types.

File: application/models/ledgerreport.go [hunk 2]
```diff
@@ -91,14 +91,25 @@ func (lr *LedgerReport) IsActorAllowedTo(tx *pop.Connection, actor User, perm Pe
...
 	lr.LoadFile(tx, false)
+	lr.LoadLedgerEntries(tx, false)
+
+	transactionCount := len(lr.LedgerEntries)
+	unclearedCount := 0
+	for _, e := range lr.LedgerEntries {
+		if !e.DateEntered.Valid {
+			unclearedCount++
+		}
```
Comment (Baggerone) [inline application/models/ledgerreport.go:101]: Would it be OK to add a `break` statement here? 
Or were you planning to use the full `unclearedCount` somehow?

**Review by Baggerone — APPROVED** [review]
A couple of suggestions.

File: application/models/ledgerreport.go (hunk 1 above)
Comment (Schparky) [reply application/models/ledgerreport.go:92]: I think that may be an artifact of using Go's "implement function" feature. I sometimes forget to give it a specific return type. Thanks for drawing it to my attention.

File: application/models/ledgerreport.go (hunk 2 above)
Comment (Schparky) [reply application/models/ledgerreport.go:102]: Good one. I'll even change the variable to a bool instead of an int.

**Pull request merged**